### PR TITLE
Corrects PAR and FLORT SN for D00002

### PR DESCRIPTION
**Pull request by cdobs**

### CODE REVIEW

File: deployment/CP05MOAS-GL336_Deploy.csv [hunk 1]
```diff
@@ -1,55 +1,55 @@
...
+SI20150110,,MM20150327,,CP05MOAS-GL336-03-CTDGVM000,1,1,2015-01-11T20:57:00,2015-03-27T21:18:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00336,CGINS-CTDGVM-09030,40.02043,-70.38543,,1000,1000,FZ-1,,
+SI20150110,,MM20150327,,CP05MOAS-GL336-04-DOSTAM000,1,1,2015-01-11T20:57:00,2015-03-27T21:18:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00336,CGINS-DOSTAM-00117,40.02043,-70.38543,,1000,1000,FZ-1,,
+SI20150110,,MM20150327,,CP05MOAS-GL336-05-PARADM000,1,1,2015-01-11T20:57:00,2015-03-27T21:18:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00336,CGINS-PARADM-50152,40.02043,-70.38543,,1000,1000,FZ-1,,
+CT20160808,,TS20160818,,CP05MOAS-GL336-00-ENG000000,2,1,2016-08-09T13:54:00,2016-08-18T12:45:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00336,CGCON-GLDRCP-00336,39.84253,-70.34048,,1000,1000,FZ-1,,
+CT20160808,,TS20160818,,CP05MOAS-GL336-01-ADCPAM000,2,1,2016-08-09T13:54:00,2016-08-18T12:45:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00336,CGINS-ADCPAM-43106,39.84253,-70.34048,,1000,1000,FZ-1,,
+CT20160808,,TS20160818,,CP05MOAS-GL336-02-FLORTM000,2,1,2016-08-09T13:54:00,2016-08-18T12:45:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00336,CGINS-FLORTM-04194,39.84253,-70.34048,,1000,1000,FZ-1,,
```
Comment (snwhite) [inline deployment/CP05MOAS-GL336_Deploy.csv:10]: Looks correct based on calibration date in Glider 336 Configuration Record, and autoexec record in 3304-00150-00054 FCP.

File: deployment/CP05MOAS-GL336_Deploy.csv [hunk 2]
```diff
@@ -1,55 +1,55 @@
...
+CT20160808,,TS20160818,,CP05MOAS-GL336-03-CTDGVM000,2,1,2016-08-09T13:54:00,2016-08-18T12:45:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00336,CGINS-CTDGVM-09030,39.84253,-70.34048,,1000,1000,FZ-1,,
+CT20160808,,TS20160818,,CP05MOAS-GL336-04-DOSTAM000,2,1,2016-08-09T13:54:00,2016-08-18T12:45:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00336,CGINS-DOSTAM-00117,39.84253,-70.34048,,1000,1000,FZ-1,,
+CT20160808,,TS20160818,,CP05MOAS-GL336-05-PARADM000,2,1,2016-08-09T13:54:00,2016-08-18T12:45:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00336,CGINS-PARADM-50220,39.84253,-70.34048,,1000,1000,FZ-1,,
```
Comment (snwhite) [inline deployment/CP05MOAS-GL336_Deploy.csv:13]: Looks correct based on calibration date in Glider 336 Configuration Record, and autoexec record in 3304-00150-00054 FCP.

File: deployment/CP05MOAS-GL336_Deploy.csv [hunk 3]
```diff
@@ -1,55 +1,55 @@
...
+SI20150110,,MM20150327,,CP05MOAS-GL336-02-FLORTM000,1,1,2015-01-11T20:57:00,2015-03-27T21:18:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00336,CGINS-FLORTM-02858,40.02043,-70.38543,,1000,1000,FZ-1,,
+SI20150110,,MM20150327,,CP05MOAS-GL336-03-CTDGVM000,1,1,2015-01-11T20:57:00,2015-03-27T21:18:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00336,CGINS-CTDGVM-09030,40.02043,-70.38543,,1000,1000,FZ-1,,
+SI20150110,,MM20150327,,CP05MOAS-GL336-04-DOSTAM000,1,1,2015-01-11T20:57:00,2015-03-27T21:18:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00336,CGINS-DOSTAM-00117,40.02043,-70.38543,,1000,1000,FZ-1,,
+SI20150110,,MM20150327,,CP05MOAS-GL336-05-PARADM000,1,1,2015-01-11T20:57:00,2015-03-27T21:18:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00336,CGINS-PARADM-50152,40.02043,-70.38543,,1000,1000,FZ-1,,
+CT20160808,,TS20160818,,CP05MOAS-GL336-00-ENG000000,2,1,2016-08-09T13:54:00,2016-08-18T12:45:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00336,CGCON-GLDRCP-00336,39.84253,-70.34048,,1000,1000,FZ-1,,
+CT20160808,,TS20160818,,CP05MOAS-GL336-01-ADCPAM000,2,1,2016-08-09T13:54:00,2016-08-18T12:45:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00336,CGINS-ADCPAM-43106,39.84253,-70.34048,,1000,1000,FZ-1,,
```
Comment (snwhite) [inline deployment/CP05MOAS-GL336_Deploy.csv:9]: Configuration record and autoexec file from 3304-00150-00054 FCP say DVL assy 0550.  Config record says sensor S/N is 649696.

Comment (rtravis-whoi) [reply deployment/CP05MOAS-GL336_Deploy.csv:9]: agree - config record 'Unit_336_GCR_3177_2016-05-24' in Vault lists bay assy SN 0550, w/ ADCP sensor SN 649696

File: deployment/CP05MOAS-GL336_Deploy.csv (hunk 1 above)
Comment (rtravis-whoi) [reply deployment/CP05MOAS-GL336_Deploy.csv:10]: good

File: deployment/CP05MOAS-GL336_Deploy.csv (hunk 2 above)
Comment (rtravis-whoi) [reply deployment/CP05MOAS-GL336_Deploy.csv:13]: good

**Review by rtravis-whoi — COMMENTED** [review]
agree w Sheri - config record 'Unit_336_GCR_3177_2016-05-24' in Vault lists bay assy SN 0550, w/ ADCP sensor SN 649696.  The other 2 updates look good.

File: deployment/CP05MOAS-GL336_Deploy.csv [hunk 4]
```diff
@@ -1,55 +1,55 @@
...
+SI20150110,,MM20150327,,CP05MOAS-GL336-02-FLORTM000,1,1,2015-01-11T20:57:00,2015-03-27T21:18:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00336,CGINS-FLORTM-02858,40.02043,-70.38543,,1000,1000,FZ-1,,
+SI20150110,,MM20150327,,CP05MOAS-GL336-03-CTDGVM000,1,1,2015-01-11T20:57:00,2015-03-27T21:18:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00336,CGINS-CTDGVM-09030,40.02043,-70.38543,,1000,1000,FZ-1,,
+SI20150110,,MM20150327,,CP05MOAS-GL336-04-DOSTAM000,1,1,2015-01-11T20:57:00,2015-03-27T21:18:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00336,CGINS-DOSTAM-00117,40.02043,-70.38543,,1000,1000,FZ-1,,
+SI20150110,,MM20150327,,CP05MOAS-GL336-05-PARADM000,1,1,2015-01-11T20:57:00,2015-03-27T21:18:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00336,CGINS-PARADM-50152,40.02043,-70.38543,,1000,1000,FZ-1,,
+CT20160808,,TS20160818,,CP05MOAS-GL336-00-ENG000000,2,1,2016-08-09T13:54:00,2016-08-18T12:45:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00336,CGCON-GLDRCP-00336,39.84253,-70.34048,,1000,1000,FZ-1,,
+CT20160808,,TS20160818,,CP05MOAS-GL336-01-ADCPAM000,2,1,2016-08-09T13:54:00,2016-08-18T12:45:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00336,CGINS-ADCPAM-49696,39.84253,-70.34048,,1000,1000,FZ-1,,
```
Comment (rtravis-whoi) [inline deployment/CP05MOAS-GL336_Deploy.csv:9]: ADCP SN correction looks good

**Review by rtravis-whoi — APPROVED** [review]

**Review by snwhite — APPROVED** [review]

**Pull request merged**